.vscode/settings.json

```diff
@@ -20,7 +20,9 @@
     [".toHaveClass\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"],
     [".toHaveExactClasses\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"],
     ["(?:qtMerge|qtJoin)\\(([^\\);]*)[\\);]", "[`'\"`]([^'\"`,;]*)[`'\"`]"],
-    [":\\s*?[\"'`]([^\"'`]*).*?,"]
+    [":\\s*?[\"'`]([^\"'`]*).*?,"],
+    ["cva\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"],
+    ["cx\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"]
```

Comment:
@aizad-deriv this has already been added to the main branch. Can you please take a pull from the main instead? Sorry for the hassle :pray: